sql/core/src/main/scala/org/apache/spark/sql/internal/BaseSessionStateBuilder.scala

```diff
@@ -146,6 +147,15 @@ abstract class BaseSessionStateBuilder(
     extensions.buildParser(session, new SparkSqlParser())
   }
 
+  /**
+   * Script interpreter that produces execution plan for sql batch procedural language.
+   *
+   * Note: this depends on the `conf` field.
```

Comment:
do we know what this part of the comment actually means? I see that it's being reused around, but not sure what it means exactly... I mean, I don't see that `conf` is used anywhere in `scriptingInterpreter` creation...

Reply:
Removed.